pubspec.yaml

```diff
@@ -1,15 +1,16 @@
 name: fortune_1000_companies
-description: A list of fortune_1000_companies.
 version: 0.1.0
+description: A list of fortune_1000_companies.
 author: Brady Becker <bradybeck@gmail.com>
 homepage: http://github.com/bradybeck/fortune_1000_companies
 
 environment:
-  sdk: '>=1.0.0 <2.0.0'
+  sdk: '>=1.24.3 <3.0.0'
 
 dev_dependencies:
-    coverage: ^0.7.2
-    dart_dev: ^1.1.1
-    dart_style: ^0.2.4
-    dartdoc: ">=0.8.0 <0.10.0"
-    test: ^0.12.4
+  build_runner: ">=0.6.0 <2.0.0"
```

Comment:
Are the build packages actually necessary for this, since it doesn't have any internal build processes?